tests/integration/src/test/scala/mesosphere/marathon/integration/SharedMemoryIntegrationTest.scala

```diff
@@ -23,8 +22,6 @@ class SharedMemoryIntegrationTest extends AkkaIntegrationTest with EmbeddedMarat
     imageProviders = Some("docker")
   )
 
-  override implicit lazy val patienceConfig: PatienceConfig = PatienceConfig(timeout = Span(30, Seconds), interval = Span(1, Seconds))
```

Comment:
This setting exposed 4 different bugs in Marathon and Mesos codebases. Should we be brave and use it for all IT?

Reply:
We might, as long as we use a custom timeout for the marathon startup future